### DISCUSSION TITLE
Indian Road Shields

### DISCUSSION THREAD

**Discussion by SirSireesh** [organicmaps/organicmaps#8585] [Ideas]
In India Organic Maps currently displays the generic white road shields for all highways. I made some quick changes to demo what route shields could look like for Indian highways:

![2024-06-28T19:05:13,702600104+05:30](https://github.com/organicmaps/organicmaps/assets/22617812/6c604850-471c-4d21-a271-f06f1550af7f)

Some background - India has three major road types which have route shields: National Expressways (Blue), National Highways (Yellow) and State Highways (Green). In the screenshot above I included both the generic coloured route shields and the proper backings, to show what both would look like. I'm putting this up for discussion to get feedback on if we want the generic route shields or the pictures, based on which I can then make a proper PR.

**Comment by biodranik** [comment]
Please check how current USA shields are implemented. New shields can be added in the same way.
